pyfr/__main__.py

```diff
@@ -176,23 +176,22 @@ def process_import(args):
 
 
 def process_partition_list(args):
-    with h5py.File(args.mesh, 'r') as mesh:
-        print('name', 'parts', sep=args.sep)
+    print('name', 'parts', sep=args.sep)
```

Comment:
I think we want this in the `with` statement so that if you give a bad file name you do not get a header printed out.